masschallenge/impact-api
# [AC-7518] Normalize program role and startup status names for display

https://masschallenge.atlassian.net/browse/AC-7518

To test:
testing instructions from [this PR](https://github.com/masschallenge/front-end/pull/215) should help you see users in the directory

Labels on user cards should be in title case (Each Word Capitalized), and trailing parenthesized program abbreviations like "(BOS)" should be stripped.
